odl/contrib/fom/test/test_supervised.py

```diff
@@ -6,24 +6,58 @@
 # v. 2.0. If a copy of the MPL was not distributed with this file, You can
 # obtain one at https://mozilla.org/MPL/2.0/.
 
-"""Tests for supervised FoMs."""
+"""Test for the figures of merit (FOMs) that use a known ground truth."""
 
 from __future__ import division
 import numpy as np
 import pytest
 import scipy.signal
 import scipy.misc
 import odl
-import odl.contrib.fom
-from odl.contrib.fom.util import filter_image_sep2d
-from odl.util.testutils import simple_fixture
+from odl.contrib import fom
+from odl.util.testutils import simple_fixture, noise_element
 
 fft_impl = simple_fixture('fft_impl',
```

Comment:
Don't we have a global fixture for this @kohr-h?

Reply:
Not global AFAIK